src/pages/TeamSettings/Projects.vue

```diff
@@ -120,10 +121,10 @@ export default {
         : this.headers.filter(header => header.mobile)
     },
     isReadOnlyUser() {
-      return this.tenant.role === 'READ_ONLY_USER'
+      return !this.isTenantAdmin && !this.hasPermission('create', 'project')
```

Comment:
This should be broken up to `create:project`, `update:project`, and `delete:project` checks